### DISCUSSION TITLE
How to show toolbar for all the selected components

### DISCUSSION THREAD

**Discussion by gandhar-workik** [GrapesJS/grapesjs#4045] [Q&A] [answered]
Hey @artf, I am trying to select multiple components and I want the toolbar to appear on all of them? Is this possible?

On the demo I tried doing the following but it isn't working.
`editor.getSelectedAll().forEach(comp => comp.initToolbar())`

**Comment by artf** [comment, marked as answer]
No, currently GrapesJS shows the toolbar only on the last selected component and there is no easy way to customize such behavior. Honestly, I don't think supporting multiple toolbars makes sense but in long term, I'll try to add the possibility to customize the rendered rectangle for components so it would be possible to accomplish what you're asking.

**Reply by gandhar-workik** [reply]
Alright! Thanks for the response!
